js/sketchfab.js

```diff
@@ -119,7 +120,13 @@ SketchfabPlugin.prototype.onSubmit = function onSubmit() {
 	var modelId = url.substring(url.lastIndexOf('/') + 1);
 	var text = this.renderText(modelId);
 
-	this.insertAtCursor(text);
+	if ((this.mode || '').toLowerCase() === 'wysiwyg') {
```

Comment:
We instantiate an editor using this code when we are in a wysiwyg-like mode:

`var editor = new wysihtml5.Editor($currentEditableTextarea[0], editorRules);`

This is attached to each text area in `editorInit()`. See `editor.js` bundled with the editor plugin in core. My Javascript-fu is a little weak and I'm just skimming here so apologies if that isn't too helpful. Just keep in mind that explicit `wysiwyg` is not the only possible wysiwyg mode.